lib/instana/activators/sidekiq_client.rb

```diff
@@ -5,7 +5,8 @@ module Instana
   module Activators
     class SidekiqClient < Activator
       def can_instrument?
-        defined?(::Sidekiq) && ::Sidekiq.respond_to?(:configure_client) && ::Instana.config[:'sidekiq-client'][:enabled]
+        defined?(::Sidekiq) && defined?(::Sidekiq::VERSION) && ::Sidekiq.respond_to?(:configure_client) && ::Instana.config[:'sidekiq-client'][:enabled] &&
+          Gem::Specification.find_by_name('sidekiq').version < Gem::Version.new('5.3')
```

Comment:
_question:_ Just out of curiosity: Do we ever log something if `can_instrument?` returns false? This is probably out of scope for this PR, but it might be valuable to log this at least at debug level.